runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileTagList/ProcessTileTagList.tsx

```diff
@@ -64,15 +67,25 @@ const ProcessTileTagList: FC<ProcessTileTagListProps> = ({ tags }) => {
         setIsTagBoxExpanded(!isTagBoxExpanded);
     };
 
+    const tagNameFitSearch = (tagName: string) => Boolean(tagName.match(RegExp(searchValue, 'ig')));
+
     return (
         <Container>
             <TagBox ref={refTagBox} $isExpanded={isTagBoxExpanded} >
-                {tags.map((tag) =>
-                    <Chip
-                        label={tag.name}
-                        key={tag.name}
-                        size='small'
-                    />)
+                {tags.filter(tag => tagNameFitSearch(tag.name))
+                    .concat(tags.filter(tag => !tagNameFitSearch(tag.name)))
+                    .map((tag) =>
+                        <Chip
+                            label={
+                                <HighlightText
+                                    text={tag.name}
+                                    matchingText={searchValue}
+                                    matchStyle={highlightTextStyle}
+                                />
+                            }
+                            key={tag.name}
+                            size='small'
+                        />)
                 }
```

Comment:
you can refactor this and use reduce method

`{tags
    .reduce((acc, tag) => 
      tagNameFitSearch(tag.name) 
          ? [tag, ...acc] 
          : [...acc, tag]
    , [])
    .map((tag) =>
        <Chip
            label={
                <HighlightText
                    text={tag.name}
                    matchingText={searchValue}
                    matchStyle={highlightTextStyle}
                />
            }
            key={tag.name}
            size='small'
        />)
}`

Reply:
Done.